gameplay/src/TextBox.h

```diff
@@ -112,6 +112,11 @@ class TextBox : public Label
 
     virtual void addListener(Control::Listener* listener, int eventFlags);
 
+    /**
+     * Update the text being edited.
+     */
+    void setText(char const *text) override;
```

Comment:
I agree with the benefits. This engine started long before C++ 11. However, we are looking for also consistency. Throwing random qualifier is not something I am about to do. If you really want this benefit then add a feature issue for changing all virtual usage to override and fix it. Don't propose changes in a single API to break consistency for others using it.